examples/tutorials/high_level_tutorial.py

```diff
@@ -267,7 +264,7 @@
 """
 
 # %% Create individual filenames for each system
-filenames = [f"batch_traj_{i}.h5" for i in range(len(systems))]
+filenames = [f"tmp/batch_traj_{i}.h5" for i in range(len(systems))]
```

Comment:
nit: Do you want to auto-clean up at the end of the tutorial and delete tmp?

Reply:
better not, could be people (like myself) have other stuff in `tmp/` folder which they want to keep

Reply:
could write to a subfolder in `tmp/` and clean that up instead. then only minor concern is that people might want to manually inspect the files and be annoyed if they need to comment out the removal code and rerun tutorial